package.json

```diff
@@ -14,25 +14,25 @@
   },
   "dependencies": {
     "@babel/compat-data": "^7.19.1",
-    "@bcrs-shared-components/action-chip": "1.1.1",
-    "@bcrs-shared-components/breadcrumb": "2.1.1",
-    "@bcrs-shared-components/business-lookup": "1.1.1",
-    "@bcrs-shared-components/certify": "2.1.1",
-    "@bcrs-shared-components/completing-party": "2.1.1",
+    "@bcrs-shared-components/action-chip": "1.1.3",
+    "@bcrs-shared-components/breadcrumb": "2.1.3",
+    "@bcrs-shared-components/business-lookup": "1.1.3",
+    "@bcrs-shared-components/certify": "2.1.3",
+    "@bcrs-shared-components/completing-party": "2.1.3",
     "@bcrs-shared-components/confirm-dialog": "1.2.1",
-    "@bcrs-shared-components/contact-info": "1.2.1",
+    "@bcrs-shared-components/contact-info": "1.2.3",
     "@bcrs-shared-components/corp-type-module": "1.0.8",
-    "@bcrs-shared-components/court-order-poa": "2.1.1",
-    "@bcrs-shared-components/date-picker": "1.2.1",
+    "@bcrs-shared-components/court-order-poa": "2.1.3",
+    "@bcrs-shared-components/date-picker": "1.2.3",
     "@bcrs-shared-components/detail-comment": "1.1.1",
-    "@bcrs-shared-components/enums": "1.0.23",
-    "@bcrs-shared-components/fee-summary": "1.2.1",
+    "@bcrs-shared-components/enums": "1.0.25",
```

Comment:
This is the real change here (added an enum value). The other packages depended on enums so they got upversioned, too.

Note that there's a 2-patch-version change due to a Lerna versioning thing (there was actually just 1 change).